sdk/servicebus/service-bus/samples/javascript/browseMessages.js

```diff
@@ -26,12 +26,12 @@ const queueName = process.env.QUEUE_NAME || "<queue name>";
 async function main() {
   const sbClient = new ServiceBusClient(connectionString);
   // If receiving from a subscription you can use the getReceiver(topic, subscription) overload
-  // In this case since we're using `diagnostics.peek()` that any lock mode will work.
-  // `diagnostics.peek()` does not lock messages in either lock mode.
+  // Since browsing messages doesnt take a lock on the message, the receive mode passed to getReceiver
+  // is irrelvant to this sample code
```

Comment:
```suggestion
  // Since browsing messages doesn't take a lock on the message, the receive mode passed to getReceiver
  // is irrelevant to this sample code.
```